README.md

```diff
@@ -97,7 +97,29 @@ $ ./build-all.py
 
 Please note that this script requires Python 3.
 
-## Getting code onto the device: Gemma M0
+## How to use a BSP (i.e. getting started writing your own code)
+
+To bootstrap your own project you should be able to copy/paste the Rust code from the examples folder within the folder of the BSP you've chosen. But you shouldn't copy the `Cargo.toml` file from there, since that's not only used for the examples, but also for the whole BSP itself. You want to make your own `Cargo.toml` file. If you're new to this and have no clue what you're doing then this is probably the line you want in there:
+
+```rust
+[dependencies]
+feather_m0 = {  git = "https://github.com/atsamd-rs/atsamd", features = ["unproven"]  }
```

Comment:
@nQue, maybe you could add `unproven` as a default feature for the HAL in this commit? I think that would be helpful, and perhaps a good exercise for someone new to Rust.

Separately, you might want to clarify that this pulls the BSP from `master`. I'm not sure if that's the best thing for new users. It's probably the best choice right now, given the flux of things. But it may not be the best choice in the future.

Reply:
@bradleyharden Sure I could add that, but are you sure that's a good idea? It sounds kinda fishy. Are you one of the main developers of this repo? I say it seems fishy because it would hide that they're unproven, thus kinda removing the reason for having them labeled as "unproven" to begin with. We might as well just remove the "unproven" tag from them? That sounds like a less fishy fix :)

Reply:
@bradleyharden Oh wait, now I see that you suggested it in the Matrix chat, with a motivation, and that @jbeaurivage thumbed-upped it, so then I feel confident to add it :)

Reply:
@nQue, I think your instincts are good, but this case is a bit different. `embedded-hal` never removed the `unproven` feature gate for several traits in `v0.2.0+`. Since then, they jumped to alpha releases of `v1.0`. We plan to adopt `v1.0` once it's released, but we want to wait for it to stabilize first. In the meantime, *many* features of this HAL depend on the `unproven` feature, which is frustrating. Ultimately, I think it will be easier for everyone if we make `unproven` a default feature.

@sajattack, do you agree?

Reply:
@nQue bradleyharden is one of the maintainers.
I agree that `unproven` should be standard given how unusable a lot of this HAL is without it and how long these traits have been "unproven".

While publishing the next release this example should be updated to point to crates.io instead IMO.